packages/catalog-process/src/utilities/config.ts

```diff
@@ -37,4 +36,7 @@ const Config = z
       : true
   );
 
-export const config = Config.parse(process.env);
+export const config = {
+  ...commonsConfig,
```

Comment:
Not sure I understand this, but is the log level going to be read from this object? Since the logging code is in the commons package, doesn't it read its own config? If so, is adding the `logLevel` here useful to actually control the log level or only if we initialize a new logger?

Reply:
we need to propagate it here because the logLevel is read also in app.ts to configure express-winston. Does it make sense?